MTA-6893 Document Hub OIDC authentication (#382)
migtools/mta-documentation: docs/topics/mta-install/proc_add-users-assign-roles-web-console.adoc

:_newdoc-version: 2.18.5
:_template-generated: 2025-07-31
:_mod-docs-content-type: PROCEDURE

[id="add-users-assign-roles-web-console_{context}"]
= Add users and assign roles in the web console

[role="_abstract"]

> Administrators can manage users and assign one or more roles to a user for local authentication and authorization. If you assign one or more roles to a user, the user gets all the unique permissions associated with the roles.

Reviewer: ```suggestion
As an administrator, you can manage users and assign roles to them for local authentication and authorization. When you assign roles to a user, they receive all the unique permissions that are associated with those roles.
```
Change the third-person "Administrators" to the second-person "As an administrator, you can..." to maintain an active, conversational tone.

The verb "gets" is colloquial. Replacing it with "receive" aligns better with technical writing standards.

"Assign one or more roles to a user" is repeated twice. Using the pronoun "they" for the user simplifies the second sentence.

.Prerequisites

* You are an {ProductShortName} Administrator.

> * The cluster administrator installed {ProductShortName} 8.2.

Reviewer: ```suggestion
* You are an {ProductShortName} administrator.
* The cluster administrator has installed {ProductShortName} 8.2.
```
Switched from `8.2.0` to `8.2`, can you use an attribute and be consistent with the attribute as you have used `8.2.0` in earlier versions of pre-reqs

.Procedure

> . Log in to the {ProductShortName} web console.
> . In the Administration mode, go to *User Management*.
> . Click *Users*.
> . Enter the following on the *Create user* page:
> * Username in the *Login* field.
> * Name of the user.
> * Email address of the user.
> * A new password in the *Password* field.
> * The same password in the *Confirm Password* field.
> * Select one or more roles from the *Roles* list.
> . Click *Create*.

Reviewer: ```suggestion
. Log in to the {ProductShortName} web console.
. In the *Administration* mode, click *User Management*.
. Click *Users*.
. On the *Create user* page, specify the following information:
* Enter a username in the *Login* field.
* Enter the name of the user.
* Enter the email address of the user.
* Enter a new password in the *Password* field.
* Enter the same password in the *Confirm Password* field.
* Select one or more roles from the *Roles* list.
. Click *Create*.
```

Your bulleted list mixes noun phrases ("Username in the...") with imperative verb phrases because these are instructions, every bullet point should start with an imperative verb (like "Enter" or "Select") and end with a period.
